src/timeMachine/components/TimeMachineFluxEditor.tsx

```diff
@@ -142,6 +143,17 @@ const TimeMachineFluxEditor: FC<Props> = ({
     onSetActiveQueryText(editorInstance.getValue())
   }
 
+  const handleActiveQuery = text => {
+    const currentText = activeQueryText.replace(
+      `\n  |> aggregateWindow(every: v.windowPeriod, fn: mean, createEmpty: false)\n  |> yield(name: "mean")`,
```

Comment:
so this only captures select all and rip right? not some maniac pressing the backspace a million times. or actually it would. can you limit this check to just `aggregateWindow` so that we dont count people who rename the yield or change the function?